Source: Thomas Anderson
Upstream: Relevant lines are commented out in upstream git
Reason: Add spacing so CPPFLAGS are not mistakenly appended without a space

diff -aur guvcview-src-2.0.1-orig/configure.ac guvcview-src-2.0.1/configure.ac
--- guvcview-src-2.0.1-orig/configure.ac	2014-10-05 03:41:44.000000000 -0700
+++ guvcview-src-2.0.1/configure.ac	2015-07-08 14:49:00.316232436 -0700
@@ -149,7 +149,7 @@
 
 dnl Check for avcodec.h directly
 dnl if all tests fail must add avcodec.h path to C_INCLUDE_PATH before running configure
-CPPFLAGS+="${GVIEWV4L2CORE_CFLAGS}"
+CPPFLAGS+=" ${GVIEWV4L2CORE_CFLAGS}"
 AC_CHECK_HEADERS([avcodec.h libavcodec/avcodec.h ffmpeg/avcodec.h])
 
 dnl --------------------------------------------------------------------------
